cortex/secondary/stationary_proportion.py

```diff
@@ -0,0 +1,32 @@
+from ..feature_types import secondary_feature, log
+from ..raw.accelerometer import accelerometer
+import pandas as pd
+import numpy as np
+
+MS_IN_A_DAY = 86400000
+
+
+@secondary_feature(
+    name='cortex.feature.stationary_proportion',
+    dependencies=[accelerometer]
+)
+def stationary_proportion(g=9.57, eps=.1, col='z', **kwargs):
+    """
+    Compute the proportion of "stationary" accelerometer readings.
+    An accelerometer reading is "stationary" if the z component is close to gravitational acceleration (9.81 m/s^2)
```

Comment:
@rydmel Could you add `param` docs for `g`, `eps`, and `col` (and a `return` doc)? I'm not sure if there's a great format for it already but [it seems like NumPy's convention is pretty popular](https://numpydoc.readthedocs.io/en/latest/format.html#parameters).